lumicks/pylake/detail/confocal.py

```diff
@@ -273,13 +273,39 @@ def _to_spatial(self, data):
         raise NotImplementedError
 
     @cachetools.cachedmethod(lambda self: self._cache)
-    def _image(self, channel):
-        assert channel in ("red", "green", "blue")
+    def _image(self, channel) -> np.ndarray:
+        """Returns a reconstructed image.
+
+        Note: Do *not* modify or return this image without making a copy, as it will return a
+        reference to the cached image, which makes the `ConfocalImage` mutable.
```

Comment:
```suggestion
        Warning: Do *not* modify or return this image without making a copy, as it will return a
        reference to the cached image, which makes the `ConfocalImage` mutable.
```